modules/fundamental/src/sbom/model/details.rs

```diff
@@ -149,7 +151,7 @@ impl SbomDetails {
             JOIN "version_range" ON "product_version_range"."version_range_id" = "version_range"."id" AND version_matches("product_version"."version", "version_range".*)
 
             -- now find matching purls in these statuses
-            JOIN base_purl ON "product_status"."package" LIKE CONCAT("base_purl"."namespace", '/', "base_purl"."name") OR "product_status"."package" = "base_purl"."name"
+            JOIN base_purl ON product_status.package = base_purl.name OR product_status.package LIKE CONCAT(base_purl.namespace, '/%')
```

Comment:
@JimFuller-RedHat I believe this line should be

`JOIN base_purl ON product_status.package = base_purl.name OR product_status.package LIKE CONCAT(base_purl.namespace, '/', "base_purl"."name")`

Like this it returns too many false positives.